### PR TITLE
generate.js: Display completed task number

### LINKED ISSUE #16: Display the number of tasks completed for each org
#4 only sorted.  The number of tasks completed should be shown.

### PR DESCRIPTION
**Pull request by tioirawan**
Display the number of tasks completed for each org and fix listed info

Closes https://github.com/coala/gci-leaders/issues/16

### CODE REVIEW

**Comment by yukiisbored** [comment]
ack https://github.com/coala/gci-leaders/pull/18/commits/03be0413f52f63e51c6ccc1a8bb45a86b2201ffb

**Comment by yukiisbored** [comment]
Gonna leave merge to @blazeu

File: generate.js [hunk 1]
```diff
@@ -43,11 +43,12 @@ const template = `
   </head>
   <body>
     <h1>GCI Current Leaders</h1>
-    <i>Names are listed alphabetically</i>
+    <i>Participants names are listed alphabetically</i>
```
Comment (jayvdb) [inline generate.js]: Something to consider...

These values are not normal 'names', as many are usernames.

> The leading participants for each organization are listed alphabetically according to their "display name".

?

It is a bit too wordy...

Comment (blazeu) [reply generate.js]: Maybe just insert "display" after "participants" ?

File: generate.js [hunk 2]
```diff
@@ -43,11 +43,12 @@ const template = `
   </head>
   <body>
     <h1>GCI Current Leaders</h1>
-    <i>Names are listed alphabetically</i>
+    <i>The participants are listed alphabetically using on their "display name"</i>
```
Comment (jayvdb) [inline generate.js]: `on ` doesnt belong here.  That was my bad writing.

File: generate.js [hunk 3]
```diff
@@ -43,7 +43,7 @@ const template = `
   </head>
   <body>
     <h1>GCI Current Leaders</h1>
-    <i>Participants names are listed alphabetically</i>
+    <i>The participants are listed alphabetically using on their "display name"</i>
```
Comment (blazeu) [inline generate.js]: See @jayvdb's comment again, he edited it.

**Review by blazeu — APPROVED** [review]

**Comment by blazeu** [comment]
Now, you just need to squash it into a single commit like usual.

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/18/commits/fcae75f760ab51369a0dc3170f7496b44bae07c8
